thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiLog.java

```diff
@@ -54,8 +54,28 @@ public void log(final int level, final String tag, final String msg) {
         }
     };
 
+    /**
+     * no-op version, doesn't log
+     */
+    public static Logger NOOP = new Logger() {
+        @Override
+        public void log(final int level, final String tag, final String msg) {
+            // no-op
+        }
+    };
+
     private static Logger logger;
 
+    /**
+     * forward log to {@link TiLog} for logging
+     */
+    public static Logger TI_LOG = new Logger() {
+        @Override
+        public void log(final int level, final String tag, final String msg) {
+            TiLog.log(level, tag, msg);
```

Comment:
I recomment to remove the "self login".
Instead i would add a `System.out.print(level, tag, msg)` (or smth)